src/test/java/uk/gov/hmcts/reform/bailcaseapi/domain/handlers/presubmit/UploadSignedDecisionNoticeHandlerTest.java

```diff
@@ -87,29 +89,34 @@ public void setUp() {
         when(callback.getCaseDetails()).thenReturn(caseDetails);
         when(caseDetails.getCaseData()).thenReturn(bailCase);
         when(dateProvider.nowWithTime()).thenReturn(nowWithTime);
+        when(signedDecisionNoticeMetadata1.getTag()).thenReturn(DocumentTag.SIGNED_DECISION_NOTICE);
+        when(unsignedDecisionNoticeMetadata1.getTag()).thenReturn(DocumentTag.BAIL_DECISION_UNSIGNED);
     }
 
     @Test
     void should_add_new_document_to_the_case() {
```

Comment:
please rename the test to reflect the scenario more accurately. Like add_signed_notice_remove_unsigned_notice()

Reply:
Changed name of test to should_add_signed_document_remove_unsigned_document()